src/main/java/com/github/hcsp/controlflow/Main.java

```diff
@@ -9,7 +9,15 @@
      * @param str 给定的字符串
      * @return 字符串中大写英文字母出现的次数
      */
-    public static int countUpperCaseLetters(String str) {}
+    public static int countUpperCaseLetters(String str) {
+        int i=0;
+        for (i=0;i<=str.length();i++){
```

Comment:
```suggestion
        for (i=0;i<str.length();i++){
```

字符串中的索引是从零开始的，即，对于一个长度为3的字符串，i只能取0/1/2，不能取3，所以这里等号是不行的。希望你能使用调试器单步跑一下，理解这个过程，以及为什么不能取到等号。